changelog.d/7780.misc

```diff
@@ -0,0 +1,2 @@
+Allow to use higher versions of prometheus_client which introduce no breaking
+changes. Contributed by Oliver Kurz.
```

Comment:
I think this needs to be a single line to make GitHub Markdown happy.

Also, would it be worth noting the new maximum supported version?

Reply:
sure, could be. Both suggestions done. Thx.